2020-06-04T11:26:11-0300 [-] Creating router ...
2020-06-04T11:26:11-0300 [-] Initializing Router - Number of events: 1
2020-06-04T11:26:12-0300 [-] RouterFactory starting on 8081
2020-06-04T11:26:12-0300 [-] Creating AccessPoint station ...
2020-06-04T11:26:12-0300 [-] Initializing Access Point - Number of events: 2
2020-06-04T11:26:13-0300 [stderr#error] Exception in Tkinter callback
2020-06-04T11:26:13-0300 [stderr#error] Traceback (most recent call last):
2020-06-04T11:26:13-0300 [stderr#error]   File "/usr/lib/python3.6/tkinter/__init__.py", line 1708, in __call__
2020-06-04T11:26:13-0300 [stderr#error]     return self.func(*args)
2020-06-04T11:26:13-0300 [stderr#error]   File "/home/rafael/iotfogsim/scinetsim/scinetsim/configurations.py", line 107, in <lambda>
2020-06-04T11:26:13-0300 [stderr#error]     btn_open = ttk.Button(window, text="Open Project",command=lambda:open_project(window,simulation_core))
2020-06-04T11:26:13-0300 [stderr#error]   File "/home/rafael/iotfogsim/scinetsim/scinetsim/configurations.py", line 61, in open_project
2020-06-04T11:26:13-0300 [stderr#error]     load_nodes(selected_project_name, simulation_core)
2020-06-04T11:26:13-0300 [stderr#error]   File "/home/rafael/iotfogsim/scinetsim/scinetsim/configurations.py", line 154, in load_nodes
2020-06-04T11:26:13-0300 [stderr#error]     ap.run()
2020-06-04T11:26:13-0300 [stderr#error]   File "/home/rafael/iotfogsim/scinetsim/scinetsim/standarddevice.py", line 320, in run
2020-06-04T11:26:13-0300 [stderr#error]     self.application.start()
2020-06-04T11:26:13-0300 [stderr#error]   File "/home/rafael/iotfogsim/scinetsim/applications/accesspointapp.py", line 239, in start
2020-06-04T11:26:13-0300 [stderr#error]     self.print_associated_devices()
2020-06-04T11:26:13-0300 [stderr#error]   File "/home/rafael/iotfogsim/scinetsim/applications/accesspointapp.py", line 326, in print_associated_devices
2020-06-04T11:26:13-0300 [stderr#error]     if cont >0:
2020-06-04T11:26:13-0300 [stderr#error] UnboundLocalError: local variable 'cont' referenced before assignment
2020-06-04T11:26:13-0300 [stdout#info] This divice have no simulation core instance
2020-06-04T11:26:23-0300 [-] Received SIGINT, shutting down.
2020-06-04T11:26:23-0300 [applications.routerapp.RouterFactory] (TCP Port 8081 Closed)
2020-06-04T11:26:23-0300 [-] Main loop terminated.
